.kokoro/presubmit/downstream-build.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
Are we planning to have this run as a Kokoro job or would the next step be to modify https://github.com/googleapis/gapic-generator-java/blob/main/.github/workflows/downstream-native-image.yaml to run this script?

Reply:
Kokoro. See private chat.